client/src/lib/tests/router-test.js

```diff
@@ -1,5 +1,7 @@
 /* global jest, jasmine, describe, beforeEach, it, expect, process */
-
+/* eslint-disable
+ import/extensions
+*/
 jest.dontMock('../Router.js');
```

Comment:
Maybe try a normal `import` statement and/or remove the `dontMock` statement

Reply:
update,d works as an import statement